deploy/hono-service-infinispan-device-registry-config.yml

```diff
@@ -0,0 +1,29 @@
+hono:
+  app:
+    maxInstances: 1
+    healthCheckBindAddress: 0.0.0.0
+    healthCheckPort: 8088
+  auth:
+    host: hono-service-auth.hono
+    port: 5671
+    keyPath: /workspace/oss/hono/deploy/target/deploy/helm/hono-demo-certs-jar/device-registry-key.pem
```

Comment:
looks like tied to your machine :)

Reply:
Thanks for spotting it. It was a test file, shouldn't have been committed at all.